test/Caption.spec.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Caption from '../src/Caption';
+
+describe('<Caption />', () => {
+  let wrapper = shallow(
+    <Caption placement="left">
```

Comment:
what happens if we dont supply placement prop?

Reply:
It defaults to `center`

```javascript
Caption.defaultProps = {
  placement: 'center'
};
```